src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -634,12 +634,27 @@ def merge(self,
     # TODO(flowerhack): Integrate some notion of a merge timeout.
     self._push_corpora_from_host_to_target(corpus_directories)
 
+    target_tmp_dir = None
+    if tmp_dir:
+      # Tmp dir may have artifacts, e.g. merge control file for two step merge.
+      target_tmp_dir = self._corpus_target_subdir(os.path.basename(tmp_dir))
+      self.fuzzer.device.rm(target_tmp_dir, recursive=True)
+      self.fuzzer.device.store(tmp_dir, target_tmp_dir)
+      additional_args = copy.copy(additional_args)
+      for i, argument in enumerate(additional_args):
```

Comment:
It's weird to me that `additional_args=None` in this function when it will exception if it is actually None.
Is this just to support the API?

Reply:
That's weird indeed. All other methods are doing:

```
    additional_args = copy.copy(additional_args)
    if additional_args is None:
      additional_args = []
```

let me do it here for consistency and in a few other places where it's missing